examples/Cpp/ExampleBase/ExampleBase.cpp

```diff
@@ -230,7 +230,7 @@ void ExampleBase::Run()
         }
 
         // Draw current frame
-        OnDrawFrame();
+        context->DrawNow([=]{OnDrawFrame();});
```

Comment:
It appears to me we can fix this by simple wrapping this call in an `@autoreleasepool` like this:
```cpp
#ifdef LLGL_PLATFORM_MACOS
@autoreleasepool {
    OnDrawFrame();
}
#else
OnDrawFrame();
#endif
```
It would of course need to be in a separate file such as `ExampleBaseFrame.mm` to provide Obj-C compatibility.

Reply:
You're right, I'm over-engineering this based on past experience. In similar projects I've always run into issues when drawing outside of an OS-issued update event. There's always some bit of bookkeeping to be done before and after the drawing operation to make this work reliably e.g. correctly managing HDC's on WIN32 or autorelease pools on Mac. I've also needed a call like DrawNow in the past when working with multiple drawing contexts in the same window (something I'm looking into with LLGL). But for now it's overkill for the somewhat artificial use case of these sample programs.

A more localized fix is to tweak the .cpp file and then alter the CMakeLists.txt file to force it to be compiled as Objective-c++. Then you won't need a separate .mm file for the Mac. I'll look into that.

Reply:
> A more localized fix is to tweak the .cpp file and then alter the CMakeLists.txt file to force it to be compiled as Objective-c++
That's also an option, but I'm trying to keep the CMake files simple. I worked with projects where the CMake script executes Python scripts and other indirections.